src/app/shared/model/audio-version.model.ts

```diff
@@ -127,7 +127,7 @@ export class AudioVersionModel extends BaseModel {
   }
 
   changed(field?: string | string[], includeRelations = true): boolean {
-    if (this.isNew && this.files.length === 0) {
+    if (this.isNew && this.files.length === 0 && this.explicit === undefined) {
```

Comment:
This also has the effect of prompting the user on story edit that they have unsaved changes if they have set Clean/Explicit and nothing else, which seems correct to me.